packages/ui-components/src/components/radio-list-item/radio-list-item.helper.tsx

```diff
@@ -0,0 +1,29 @@
+import { JSX, h } from '@stencil/core';
+import { LINK_MATCHER_REGEX } from './radio-list-item.config';
+import { EAnchorTarget } from '../../types';
+
+export const buildDescription = (description?: string): JSX.Element[] => {
+	if (!description) return;
+
+	const linkMatches = [...description.matchAll(LINK_MATCHER_REGEX)];
+	const parsedJSX: JSX.Element[] = [];
+	let curIdx = 0;
+
+	for (const match of linkMatches) {
+		const beforeLink = description.slice(curIdx, match.index);
+		if (beforeLink.length > 0) {
```

Comment:
It is a problem if we push an empty string for `parsedJSX`?

Reply:
as parsedJSX is initialized as an empty array, if you pass an empty string the return value will be an empty array which will not render anything